File: liveroom-libs/comp_avparam/src/main/java/com/qlive/avparam/QAudioFrameListener.java
package com.qlive.avparam;

import java.nio.ByteBuffer;

public interface QAudioFrameListener {
    void onAudioFrameAvailable(
            ByteBuffer srcBuffer,
            int size,
            int bitsPerSample,
            int sampleRate,
            int numberOfChannels
    );
}

File: liveroom-libs/comp_avparam/src/main/java/com/qlive/avparam/QAudioFrameListener.kt
package com.qlive.avparam

import java.nio.ByteBuffer

interface QAudioFrameListener {
    fun onAudioFrameAvailable(
        srcBuffer: ByteBuffer,
        size: Int,
        bitsPerSample: Int,
        sampleRate: Int,
        numberOfChannels: Int
    )
}